backend/src/main/java/com/now/naaga/placestatistics/repository/PlaceStatisticsRepository.java

```diff
@@ -1,7 +1,11 @@
 package com.now.naaga.placestatistics.repository;
 
-import com.now.naaga.placestatistics.PlaceStatistics;
+import com.now.naaga.placestatistics.domain.PlaceStatistics;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.Optional;
+
 public interface PlaceStatisticsRepository extends JpaRepository<PlaceStatistics, Long> {
+
+    Optional<PlaceStatistics> findByPlaceId(Long placeId);
 }
```

Comment:
P5: 
PlaceStatisticsService에서 subtractLike()를 사용할 때, 동시성 문제가 발생할 수 있습니다.
사소한 문제이긴 하지만, 아래와 같은 두가지 해결책이 있을 것 같습니다.
1. 비관적 락을 사용하는 방법(트랜잭션이 충돌이 발생한다고 가정하고 락을건다)
= selet ~ from ~ for update
```suggestion
    @Lock(LockModeType.PESSIMISTIC_WRITE)
    Optional<PlaceStatistics> findByPlaceId(Long placeId);
}
```

2. 낙관적 락을 사용하는 방법 
1) 레포지토리에 특정 엔티티를 낙관적 락으로 관리하는 법
```suggestion
    @Lock(LockModeType.OPTIMISTIC)
    Optional<PlaceStatistics> findByPlaceId(Long placeId);
}
```
2) 서비스에 버전 필드 추가를 통한 낙관적 락 사용
PlaceStatistics에 @Vesrsion int version이라는 필드를 추가하면, 엔티티를 조회한 시점의 버전과 수정하고난 후의 버전이 다르면 예외를 발생시킨다고 합니다.

Reply:
장소가 다양하기도 하고(사용자수도 적고), 각 장소에 비슷한 시간대에 도착해서 동일한 시간에 좋아요를 누를 확률은 0에 가까워서.. 락을 건다면 낙관적 락이 적절해보이는 것 같아요. 만약 동시성 이슈가 걸려서 예외가 터지더라도, 일정 횟수 재시도를 걸 수도 있겠네요. PESSIMISTIC_WRITE 로 건 비관적 락은 조회 성능에 영향을 끼칠 수도 있구요.

근데 `동일한 시간에 좋아요를 누를 확률은 0에 가까워서` 라는 전제 자체가 어쩌면 동시성 이슈가 발생하지 않는게 아닐까...? 하는 생각도 드네요 🤔

Reply:
일다 제 생각에도
저희가 좋아요 통계와 같은 도메인을 다루면서 동시성 고려를 하고 있다는 것은 좋으나,
일단 저희앱이 사용자가 많지 않은 상태이고 동시성을 이 PR안에서 다루는 것보다는 해당 이슈가 진짜로 발생했을 때(어? 이거 머피의 법칙인가) 혹은 그 문제가 우려되는 상황에서 대응하면 될 것 같습니다.
저도 리뷰 받고 많이 알아봤는데,
장소 좋아요같은 경우에는 일단 엄중한 결과를 필요로하는 도메인도 아니고, 성능적인 이점을 가져갈 필요도 있어 보이기 때문에 낙관적 락이 조금더 적절해보입니다.
일단은 적용은 하지 않겠습니다👍